lib/puppet/provider/templates/invoke_dsc_resource.ps1.erb

```diff
@@ -51,7 +51,7 @@ $invokeParams = @{
 <% end -%>
   }<% if resource.respond_to?(:dscmeta_module_version) %>
   ModuleName = @{
-    ModuleName      = <%= "\"#{resource.dscmeta_module_name}\"" %>
+    ModuleName      = <%= "\"#{vendored_modules_path}/#{resource.dscmeta_module_name}/#{resource.dscmeta_module_name}.psd1\"" %>
```

Comment:
We need to check if it is a built-in one that we are not vendoring and just use the name instead.